core/src/main/java/bisq/core/btc/setup/WalletConfig.java

```diff
@@ -458,7 +459,7 @@ private void startPeerGroup() {
             @Override
             public void onSuccess(@Nullable Object result) {
                 //completeExtensionInitiations(vPeerGroup);
-                DownloadProgressTracker tracker = downloadListener == null ? new DownloadProgressTracker() : downloadListener;
+                DownloadProgressTracker tracker = new DownloadProgressTracker();
```

Comment:
Should we still be using this progress tracker at all? It's from bitcoinj.

Reply:
I moved the `DownloadListener` to separate file and made it not extend `DownloadProgressTracker` (from bitcoinj) and it's used for monero now. This instance of `DownloadProgressTracker` is just provided to startBlockChainDownload on the next line but is not really used. 
I'd like to remove more code that depends on bitcoinj but probably in different PR? Here it's no related to the bitcoind instance running locally.

Reply:
Ok.